app/Http/Controllers/PerformanceController.php

```diff
@@ -49,7 +52,7 @@ public function list(Request $request)
         return view('performance.list', [
             'title' => 'Performances',
             'performances' => $performances,
-            'queryString' => $queryString,
+            'queryString' => $data->query ?? null,
```

Comment:
`$data` is of type array, not object so this must be changed to:
```suggestion
            'queryString' => $data['query'] ?? null,
```